### PR TITLE
Redux state managed bridge

### PR DESCRIPTION
**Pull request by abtestingalpha**
null
a472adcaf648c31a491dcb23cb3e434043b632e0: [synapse-interface preview link ](https://sanguine-synapse-interface-dntv2xx1g-synapsecns.vercel.app)
a4baa92b7b6c6631952280692fcacd7a9fc325df: [synapse-interface preview link ](https://sanguine-synapse-interface-g2pv15v4w-synapsecns.vercel.app)
48a6325a52aa0c45ebd1cbcffacf8abfafe79591: [synapse-interface preview link ](https://sanguine-synapse-interface-7encgcufy-synapsecns.vercel.app)
1c7bd9f96207628a87837338bd8f2d99e944c3cc: [synapse-interface preview link ](https://sanguine-synapse-interface-fqegme6zs-synapsecns.vercel.app)
0573b8619954796cb73d48a65fdaaa186680963f: [synapse-interface preview link ](https://sanguine-synapse-interface-raht8ae5a-synapsecns.vercel.app)
fe16d4ba182c8725a893a0181a6a6808afdfb5e7: [synapse-interface preview link ](https://sanguine-synapse-interface-o9dakifek-synapsecns.vercel.app)
8ad197784a70ab9ca9166788f2fd97fde32f7db3: [synapse-interface preview link ](https://sanguine-synapse-interface-hvmcqy4zl-synapsecns.vercel.app)
383e614c3481db2583bc2c8912c1df7642ff416f: [synapse-interface preview link ](https://sanguine-synapse-interface-h2rp2qr2w-synapsecns.vercel.app)
1c2a898a959d7cb6b629d9bddf133afb80645e18: [synapse-interface preview link ](https://sanguine-synapse-interface-846la6ihu-synapsecns.vercel.app)
2c2cfdf0221b95020fe8002472b3534ec9039861: [synapse-interface preview link ](https://sanguine-synapse-interface-fk1h6oxcq-synapsecns.vercel.app)
686b375b6539204d0ec5fca11e8aec21e30761bc: [synapse-interface preview link ](https://sanguine-synapse-interface-9gsvs3jpz-synapsecns.vercel.app)
6d262205b1ea198f7dbb7393f1912ffa59603a90: [synapse-interface preview link ](https://sanguine-synapse-interface-gdju169zt-synapsecns.vercel.app)
5ff8ebf04a350f3e415cbf7a918951b694aadd15: [synapse-interface preview link ](https://sanguine-synapse-interface-51esyjopz-synapsecns.vercel.app)
b792f031b18cd35fe9ca2c6fb39e9098c21084ab: [synapse-interface preview link ](https://sanguine-synapse-interface-bmdsmno4m-synapsecns.vercel.app)
af5ae7370545c4689e9fd4c66a1d8d883d0737fb: [synapse-interface preview link ](https://sanguine-synapse-interface-1ni25gequ-synapsecns.vercel.app)
d438ec2e5ba9f5935e22afe40a0a5ab48e152461: [synapse-interface preview link ](https://sanguine-synapse-interface-5mdtzbtle-synapsecns.vercel.app)
727e88b40b534efbb89142b935d672eaa2dbc520: [synapse-interface preview link ](https://sanguine-synapse-interface-b1zzo02hp-synapsecns.vercel.app)
cd12d8369adbde4b30996073199a2abcbad34c26: [synapse-interface preview link ](https://sanguine-synapse-interface-7ywcos5ym-synapsecns.vercel.app)
de03cb2afd78c7ceaaa476a4c738aed6cb66d4d6: [synapse-interface preview link ](https://sanguine-synapse-interface-m9c4ozzct-synapsecns.vercel.app)
2828857308370f49f7c2ebaa6e3f19bca30049d6: [synapse-interface preview link ](https://sanguine-synapse-interface-mx5jstcd3-synapsecns.vercel.app)
d451155f3d1936a38a7d52ed277410d24cb32eee: [synapse-interface preview link ](https://sanguine-synapse-interface-8vugp0wyn-synapsecns.vercel.app)
109d2993e3ecbb8414db963cf54c1c109287698b: [synapse-interface preview link ](https://sanguine-synapse-interface-7oombr462-synapsecns.vercel.app)
a1bbb9c865c9859b458684bf8067d6d70f2742b3: [synapse-interface preview link ](https://sanguine-synapse-interface-a5smuwhl4-synapsecns.vercel.app)
47b974d5bbbce5a4438c4145e297396c4db51b28: [synapse-interface preview link ](https://sanguine-synapse-interface-onkj60s86-synapsecns.vercel.app)
f923b49b45f6371f89c9071f48b098aa0cc4d126: [synapse-interface preview link ](https://sanguine-synapse-interface-9hl270j62-synapsecns.vercel.app)
170ce52de69309650fc1f11b0552be6141a3eb69: [synapse-interface preview link ](https://sanguine-synapse-interface-mhc68hmho-synapsecns.vercel.app)
fee15e97d39741009d558ab012c641daedd21c03: [synapse-interface preview link ](https://sanguine-synapse-interface-ma9fhytmm-synapsecns.vercel.app)
f9892e197cb8e2fd7ca0a1547a167bcf2d53462e: [synapse-interface preview link ](https://sanguine-synapse-interface-nv0cpzi3j-synapsecns.vercel.app)
6e71f9524ebd1e8d4440a4b2333efb66f9aa060d: [synapse-interface preview link ](https://sanguine-synapse-interface-eijirbp3o-synapsecns.vercel.app)
43c083a6816e997463063f80a125b66907d5ac80: [synapse-interface preview link ](https://sanguine-synapse-interface-o9zjj073u-synapsecns.vercel.app)
5a2dda8ebce2ae6339ef4e7fc736e29c5b4e54af: [synapse-interface preview link ](https://sanguine-synapse-interface-5cq7m45vr-synapsecns.vercel.app)
71e540b72ac0ef75abf69419bb2626bd5801d1d0: [synapse-interface preview link ](https://sanguine-synapse-interface-4nexxgb71-synapsecns.vercel.app)
fecc476700533f2d2f134e48c9e3463ee4eff9fb: [synapse-interface preview link ](https://sanguine-synapse-interface-h7ru060eo-synapsecns.vercel.app)
fccad2b6becea75c65f18e20a0611d18767103e8: [synapse-interface preview link ](https://sanguine-synapse-interface-6xgsuzf8f-synapsecns.vercel.app)
dc14a1f7abf27a6c6c595d6672e79aa3c310fefe: [synapse-interface preview link ](https://sanguine-synapse-interface-i1oplh2oi-synapsecns.vercel.app)
b477598b17b548c10c12927480500bbf7dbbb069: [synapse-interface preview link ](https://sanguine-synapse-interface-r8kwrjexz-synapsecns.vercel.app)
b3ff6467e7e296cc092253154fcb357f2721b2c0: [synapse-interface preview link ](https://sanguine-synapse-interface-a9mnuzlqd-synapsecns.vercel.app)
0a97ba34590cd8d559477ffa9fcc3f25ff72808f: [synapse-interface preview link ](https://sanguine-synapse-interface-4ef6wt3p9-synapsecns.vercel.app)
6726a699cf696c90b43333b7a53b337773282bd9: [synapse-interface preview link ](https://sanguine-synapse-interface-bihf540b8-synapsecns.vercel.app)
a5772d431c6124aca333f56f39e1ef282f6916c8: [synapse-interface preview link ](https://sanguine-synapse-interface-hnoq9wuqe-synapsecns.vercel.app)
099a2c83c9f9472e8db0f166e06fb8102291f39a: [synapse-interface preview link ](https://sanguine-synapse-interface-i7bh2tb77-synapsecns.vercel.app)
c155ee88411b141e1101a7012a1fc28063a12c0b: [synapse-interface preview link ](https://sanguine-synapse-interface-e6l75iasg-synapsecns.vercel.app)
e7f6e5eb1f52176643ebac46d9702e8beb33704b: [synapse-interface preview link ](https://sanguine-synapse-interface-pa1005abc-synapsecns.vercel.app)
7c138c6595cd399175cd01e0bd85cb8a9fb24a83: [synapse-interface preview link ](https://sanguine-synapse-interface-nqrnyb6s6-synapsecns.vercel.app)
c0f5b17606b16ad84502d2b2df84c4a329db1d65: [synapse-interface preview link ](https://sanguine-synapse-interface-6t2dyq54x-synapsecns.vercel.app)
61ef25965b487a789d5a35d9920d8f886a0b524c: [synapse-interface preview link ](https://sanguine-synapse-interface-r9qlhnx1f-synapsecns.vercel.app)
63f0a634aef3117451d91b7bce301fa396c135de: [synapse-interface preview link ](https://sanguine-synapse-interface-cr69yx1us-synapsecns.vercel.app)
c823738faa86ec368020ae3a7cf2a008c3036c17: [synapse-interface preview link ](https://sanguine-synapse-interface-lq4hfup3v-synapsecns.vercel.app)
8872bd0ef93224f3b8df3893c6ab1ed84bda7d58: [synapse-interface preview link ](https://sanguine-synapse-interface-74dxkd0qt-synapsecns.vercel.app)
4141a34ba6df2006580bd0ba996e01d7db4647dc: [synapse-interface preview link ](https://sanguine-synapse-interface-7ooekz4rp-synapsecns.vercel.app)
86d384930ad57e3835d0b002095fc6b5d9be8a35: [synapse-interface preview link ](https://sanguine-synapse-interface-z4j3u4pyn-synapsecns.vercel.app)
592e288afec27453e610721676593e6e92820eca: [synapse-interface preview link ](https://sanguine-synapse-interface-fmcu41suz-synapsecns.vercel.app)
18571e9a8def7a0b7ed613052015ed3f014ff097: [synapse-interface preview link ](https://sanguine-synapse-interface-q5olwmcku-synapsecns.vercel.app)
4bf7a1ed674d0a6c3d76ce1bc29a368b10f63cef: [synapse-interface preview link ](https://sanguine-synapse-interface-axwfb257x-synapsecns.vercel.app)
e8bc2484851ffe1f99c502775c3437f4f0592f6e: [synapse-interface preview link ](https://sanguine-synapse-interface-2v566vt88-synapsecns.vercel.app)

### CODE REVIEW

File: packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ import BridgeEvent from './BridgeEvent'
 import { BridgeWatcherTx } from '@types'
 import { GETLOGS_SIZE, GETLOGS_REQUEST_COUNT } from '@constants/bridgeWatcher'
 import { useSynapseContext } from '@/utils/providers/SynapseProvider'
+import { useSelector } from 'react-redux';
+import { RootState } from '../../../store/store'
```
Comment (abtestingalpha) [inline packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx:16]: we've got this alias'd:

```
import { RootState } from '@/store/store'
```

File: packages/synapse-interface/slices/bridgeSlice.ts [hunk 2]
```diff
@@ -93,6 +95,12 @@ export const bridgeSlice = createSlice({
     setDestinationAddress: (state, action: PayloadAction<Address | null>) => {
       state.destinationAddress = action.payload
     },
+    addBridgeTxHash: (state, action: PayloadAction<string>) => {
+      state.bridgeTxHashes = [...state.bridgeTxHashes, action.payload];
+    },
+    setBridgeTxHashes: (state, action: PayloadAction<string[]>) => {
```
Comment (abtestingalpha) [inline packages/synapse-interface/slices/bridgeSlice.ts:101]: can remove this if we don't use it / export it

Comment (aureliusbtc) [reply packages/synapse-interface/slices/bridgeSlice.ts]: #2497da4d9330ab7886d18fb6f93e1f211caee9e2

File: packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx (hunk 1 above)
Comment (aureliusbtc) [reply packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx]: https://github.com/synapsecns/sanguine/pull/1046/commits/2497da4d9330ab7886d18fb6f93e1f211caee9e2

**Pull request merged**